azuracast/liquidsoap_custom.liq

```diff
@@ -1,488 +1,512 @@
+    ({1w and 13h30m-23h30m}, playlist_off_air_ambient),                        #ambient: Who Say - Infinite
+    ({2w and 12h-13h58m}, once(playlist_lahmaloudclouds)),
+    ({2w and 15h30m-16h58m}, once(playlist_melyvagas)),
+    ({2w and 17h-18h58m}, once(playlist_radical_happiness)),
+    ({2w and 18h-18h58m}, once(playlist_mmn_radio)),
+    ({2w and 19h-19h58m}, once(playlist_mosolyszunet)),
+    ({2w and 20h-20h58m}, once(playlist_havizaj)),
+    ({2w and 21h-21h58m}, once(playlist_turmeric_acid)),
+    ({2w and 22h-22h58m}, once(playlist_exit)),
+    ({2w and 16h-22h30m}, playlist_off_air_ambient),                        #ambient: Melyvagas - Exit
+    ({3w and 12h-13h58m}, once(playlist_lahmacun_presents)),           #Wednesday
+    ({3w and 15h-15h58m}, once(playlist_erto_hallgatas)),
+    ({3w and 16h-16h58m}, once(playlist_erdenklang)),
+    ({3w and 17h-17h58m}, once(playlist_cleptorama)),
+    ({3w and 18h-18h58m}, once(playlist_temporary_nites)),
+    ({3w and 19h-19h58m}, once(playlist_rambo)),
+    ({3w and 20h-20h58m}, once(playlist_boombap)),
+    ({3w and 21h-21h58m}, once(playlist_eastern_daze)),
+    ({3w and 23h-23h58m}, once(playlist_linear_systems_with_gestalt)),
+    ({3w and 15h30m-23h30m}, playlist_off_air_ambient),                     #ambient: EH - Gestalt
+    ({4w and 12h-13h58m}, once(playlist_ritka_csut)),                  #Thursday
+    ({4w and 14h-14h58m}, once(playlist_kigyotanc)),
+    ({4w and 15h-15h58m}, once(playlist_btob)),
+    ({4w and 16h-16h58m}, once(playlist_fkse)),
+    ({4w and 17h-18h58m}, once(playlist_lazy_calm_raga)),
+    ({4w and 19h-19h58m}, once(playlist_dalmata_gergo_show)),
+    ({4w and 20h-20h58m}, once(playlist_keygen)),
+    ({4w and 21h-22h58m}, once(playlist_schmerz)),
+    ({4w and 4w23h-5w00h28m}, once(playlist_gyogyfurdo)),
+    ({4w and 12h30m-23h30m}, playlist_off_air_ambient),                        #ambient: Ritka csut - Gyogyf
+    ({5w and 12h-13h58m}, once(playlist_ritka_pentek)),                #Friday
+    ({5w and 14h30m-15h58m}, once(playlist_szusivacsi)),
+    ({5w and 16h-16h58m}, once(playlist_randomcsunyalanyok)),           
+    ({5w and 17h-17h58m}, once(playlist_dead_hound)),
+    ({5w and 18h-18h58m}, once(playlist_graveyard_slot)),
+    ({5w and 19h-19h58m}, once(playlist_brainfuq)),
+    ({5w and 20h-20h58m}, once(playlist_moneyka)),
+    ({5w and 21h-22h58m}, once(playlist_arcadeok_alatt)),       
+    ({5w and 12h30m-22h30m}, playlist_off_air_ambient),                        #ambient: Ritka pentek - Arcede-ok
+    ({6w and 11h30m-13h28m}, once(playlist_merites)),                  #Saturday
+    ({6w and 13h30m-14h58m}, once(playlist_kerodzo)),     
+    ({6w and 15h-17h58m}, once(playlist_torso)),
+    ({6w and 18h-19h58m}, once(playlist_rnr666)),
+    ({6w and 20h-21h58m}, once(playlist_mood_sequence)),
+    ({6w and 6w23h-7w00h28m}, once(playlist_d23)),
+    ({6w12h-7w00h}, playlist_off_air_ambient),                     #ambient: Merites - Gestalt
```

Comment:
Yeah, the label was wrong. Fixed it. For the listening experience, it would probably better to stop playing gap-filling ambient after Mood, cause there is a one hour empty slot until D23 and it's a Saturday ;)